#! /bin/sh /usr/share/dpatch/dpatch-run
## 05sans_font.dpatch by  <bernat@zoro.luffy.cx>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use free Bitstream Vera Sans instead of Tahoma

@DPATCH@

--- xrdp-0.4.0.orig/xrdp/xrdp_login_wnd.c	2007/05/06 06:50:45	1.37
+++ xrdp-0.4.0/xrdp/xrdp_login_wnd.c	2007/07/03 21:57:52	1.38
@@ -132,7 +132,7 @@
   struct xrdp_bitmap* but;
 
   /* create help screen */
-  help = xrdp_bitmap_create(300, 300, wnd->wm->screen->bpp,
+  help = xrdp_bitmap_create(340, 300, wnd->wm->screen->bpp,
                             WND_TYPE_WND, wnd->wm);
   list_insert_item(wnd->wm->screen->child_list, 0, (long)help);
   help->parent = wnd->wm->screen;
@@ -149,7 +149,7 @@
   list_insert_item(help->child_list, 0, (long)but);
   but->parent = help;
   but->owner = help;
-  but->left = 120;
+  but->left = 140;
   but->top = 260;
   but->id = 1;
   but->tab_stop = 1;
@@ -261,7 +261,7 @@
       if (g_strncmp("ask", value, 3) == 0)
       {
         /* label */
-        b = xrdp_bitmap_create(60, 20, self->screen->bpp,
+        b = xrdp_bitmap_create(70, 20, self->screen->bpp,
                                WND_TYPE_LABEL, self);
         list_insert_item(self->login_window->child_list, insert_index,
                               (long)b);
@@ -281,7 +281,7 @@
         insert_index++;
         b->parent = self->login_window;
         b->owner = self->login_window;
-        b->left = 220;
+        b->left = 230;
         b->top = 60 + 25 * count;
         b->id = 100 + 2 * count + 1;
         b->pointer = 1;
@@ -491,7 +491,7 @@
   list_add_item(self->login_window->child_list, (long)combo);
   combo->parent = self->login_window;
   combo->owner = self->login_window;
-  combo->left = 220;
+  combo->left = 230;
   combo->top = 35;
   combo->id = 6;
   combo->tab_stop = 1;
--- xrdp-0.4.0.orig/xrdp/xrdp_font.c	2007/01/12 04:47:18	1.9
+++ xrdp-0.4.0/xrdp/xrdp_font.c	2007/07/03 07:17:00	1.10
@@ -79,7 +79,7 @@
   self->wm = wm;
   make_stream(s);
   init_stream(s, 8192 * 2);
-  fd = g_file_open("Tahoma-10.fv1");
+  fd = g_file_open("sans-10.fv1");
   if (fd != -1)
   {
     b = g_file_read(fd, s->data, 8192 * 2);
--- xrdp-0.4.0.orig/xrdp/Makefile	2007-07-11 09:29:59.000000000 +0200
+++ xrdp-0.4.0/xrdp/Makefile	2007-07-11 09:30:20.000000000 +0200
@@ -35,7 +35,7 @@
 	install xrdp256.bmp $(DESTDIR)/xrdp256.bmp
 	install cursor0.cur $(DESTDIR)/cursor0.cur
 	install cursor1.cur $(DESTDIR)/cursor1.cur
-	install Tahoma-10.fv1 $(DESTDIR)/Tahoma-10.fv1
+	install sans-10.fv1 $(DESTDIR)/sans-10.fv1
 	install xrdp.ini $(CFGDIR)/xrdp.ini
 	install rsakeys.ini $(CFGDIR)/rsakeys.ini
 	install xrdp $(DESTDIR)/xrdp
@@ -45,7 +45,7 @@
 	install xrdp256.bmp $(DESTDIRDEB)/usr/lib/xrdp/xrdp256.bmp
 	install cursor0.cur $(DESTDIRDEB)/usr/lib/xrdp/cursor0.cur
 	install cursor1.cur $(DESTDIRDEB)/usr/lib/xrdp/cursor1.cur
-	install Tahoma-10.fv1 $(DESTDIRDEB)/usr/lib/xrdp/Tahoma-10.fv1
+	install sans-10.fv1 $(DESTDIRDEB)/usr/lib/xrdp/sans-10.fv1
 	install xrdp.ini $(DESTDIRDEB)/etc/xrdp/xrdp.ini
 	install rsakeys.ini $(DESTDIRDEB)/etc/xrdp/rsakeys.ini
 	install xrdp $(DESTDIRDEB)/usr/lib/xrdp/xrdp
--- xrdp-0.4.0.orig/xrdp/sans-10.fv1.uue	1970-01-01 01:00:00.000000000 +0100
+++ xrdp-0.4.0/xrdp/sans-10.fv1.uue	2007-07-11 08:37:14.000000000 +0200
@@ -0,0 +1,267 @@
+begin 644 sans-10.fv1
+M1DY4,4)I='-T<F5A;2!697)A(%-A;G,`````````````````"@`!````````
+M````$``0`/#__/\$````````````````````````````````````````````
+M````````$``0`/#__/\%``````````````````(``@`"``(``@`"`````@`"
+M````````````$``0`/#__/\%``````````````````4`!0`%``4`````````
+M````````````````&``0`/#__/\+``````````````````!(``"0``"0``/\
+M``"0``$@``?X``$@``$@``-``````````````````!``$`#P__S_"```````
+M````````````@`/@!)`$@`.``/``D`20`^``@`"``````!@`$`#P__S_#```
+M```````````````````&"``)$``)(``)(``&3```D@``D@`!$@`"#```````
+M```````````8`!``\/_\_PL``````````````````````<```B```@```0``
+M`H0`!$0`!"@`!A@``^0`````````````````$``0`/#__/\#````````````
+M``````0`!``$``0`````````````````````````$``0`/#__/\%````````
+M`````P`"``(`!``$``0`!``$``0``@`"``$`````````$``0`/#__/\%````
+M````````!``"``(``0`!``$``0`!``$``@`"``0`````````$``0`/#__/\'
+M``````````````````$`"2`'P`.`#6`!````````````````````&``0`/#_
+M_/\+``````````````````````!```!```!```!```?\``!```!```!```!`
+M`````````````````!``$`#P__S_!```````````````````````````````
+M``````(``@`$`````````!``$`#P__S_!0``````````````````````````
+M````!P```````````````````!``$`#P__S_!```````````````````````
+M``````````````(``@```````````!``$`#P__S_!``````````````````!
+M``$``@`"``(`!@`$``0`!``(``@``````!``$`#P__S_"```````````````
+M```#P`)`!"`$(`0@!"`$(`)``\```````````!``$`#P__S_"```````````
+M```````#@`"``(``@`"``(``@`"``^```````````!``$`#P__S_"```````
+M```````````#P`1@`"``(`!``(`!``(`!^```````````!``$`#P__S_"```
+M```````````````#P`0@`"``(`'``"``(`0@`\```````````!``$`#P__S_
+M"```````````````````P`%``4`"0`)`!$`'X`!``$```````````!``$`#P
+M__S_"``````````````````'P`0`!``'P`!@`"``(`1@`\```````````!``
+M$`#P__S_"``````````````````!P`(@!``%P`9@!"`$(`)@`\``````````
+M`!``$`#P__S_"``````````````````'X``@`$``0`"``(`!``$``@``````
+M`````!``$`#P__S_"``````````````````#P`0@!"`$(`/`!"`$(`0@`\``
+M`````````!``$`#P__S_"``````````````````#P`9`!"`$(`9@`Z``(`1`
+M`X```````````!``$`#P__S_!````````````````````````@`"````````
+M``(``@```````````!``$`#P__S_!````````````````````````@`"````
+M``````(``@`$`````````!@`$`#P__S_"P``````````````````````````
+M"```<``#@``$```#@```<```"``````````````````````8`!``\/_\_PL`
+M````````````````````````````````!_@`````````!_@`````````````
+M````````````&``0`/#__/\+``````````````````````````0```.```!P
+M```(``!P``.```0``````````````````````!``$`#P__S_!P``````````
+M```````#@`1``$``@`$``0````$``0```````````!@`$`#P__S_#0``````
+M````````````````^``!!@`"`@`$>0`$B0`$B0`$B@`$?``"```!!```^```
+M```````8`!``\/_\_PD``````````````````````(```4```4```B```B``
+M!_``!!``!!``"`@`````````````````&``0`/#__/\)````````````````
+M``````?@``00``00``00``?@``00``00``00``?@`````````````````!@`
+M$`#P__S_"0`````````````````````!X``"$``$```$```$```$```$```"
+M$``!X``````````````````8`!``\/_\_PH`````````````````````!^``
+M!#``!`@`!`@`!`@`!`@`!`@`!#``!^``````````````````$``0`/#__/\(
+M``````````````````?@!``$``0`!^`$``0`!``'X```````````$``0`/#_
+M_/\'``````````````````?`!``$``0`!\`$``0`!``$````````````&``0
+M`/#__/\*``````````````````````'P``((``0```0```0X``0(``0(``((
+M``'P`````````````````!@`$`#P__S_"@`````````````````````$"``$
+M"``$"``$"``'^``$"``$"``$"``$"``````````````````0`!``\/_\_P,`
+M````````````````!``$``0`!``$``0`!``$``0````````````0`!``\/_\
+M_P,`````````````````!``$``0`!``$``0`!``$``0`!``$`!@````0`!``
+M\/_\_P@`````````````````!"`$0`2`!0`&``4`!(`$0`0@```````````0
+M`!``\/_\_P<`````````````````!``$``0`!``$``0`!``$``?@````````
+M```8`!``\/_\_PL`````````````````````!@P`!@P`!10`!10`!*0`!*0`
+M!$0`!`0`!`0`````````````````&``0`/#__/\*````````````````````
+M``8(``8(``4(``2(``3(``1(``0H``08``08`````````````````!@`$`#P
+M__S_"@`````````````````````!X``"$``$"``$"``$"``$"``$"``"$``!
+MX``````````````````0`!``\/_\_P@`````````````````!\`$8`0@!"`$
+M8`?`!``$``0````````````8`!``\/_\_PH``````````````````````>``
+M`A``!`@`!`@`!`@`!`@`!`@``A```>```"```!``````````$``0`/#__/\(
+M``````````````````?`!"`$(`0@!\`$0`0@!"`$$```````````&``0`/#_
+M_/\)``````````````````````/@``80``0```8```/@```P```0``0P``/@
+M`````````````````!``$`#P__S_!P`````````````````/X`$``0`!``$`
+M`0`!``$``0```````````!@`$`#P__S_"@`````````````````````$"``$
+M"``$"``$"``$"``$"``$"``&&``!X``````````````````8`!``\/_\_PD`
+M````````````````````"`@`"`@`!!``!!```B```B```4```4```(``````
+M````````````&``0`/#__/\+``````````````````````A"``A"``1$``2D
+M``2D``*H``*H``$0``$0`````````````````!@`$`#P__S_"@``````````
+M```````````&&``"$``!(```P```P``!(``!(``"$``&&```````````````
+M```0`!``\/_\_P<`````````````````""`$0`1``H`"@`$``0`!``$`````
+M```````8`!``\/_\_PH`````````````````````!_@``!```"```$```$``
+M`(```0```@``!_@`````````````````$``0`/#__/\%```````````````'
+M``0`!``$``0`!``$``0`!``$``0`!P``````$``0`/#__/\$````````````
+M``````@`"``$``0`!``&``(``@`"``$``0``````$``0`/#__/\%````````
+M```````'``$``0`!``$``0`!``$``0`!``$`!P``````&``0`/#__/\+````
+M``````````````````#```'@``,P``88````````````````````````````
+M`````````!``$`#P__S_!P``````````````````````````````````````
+M````````#^```!``$`#P__S_!P```````````````@`!````````````````
+M`````````````````!``$`#P__S_"````````````````````````\`$(``@
+M`^`$(`1@`Z```````````!``$`#P__S_"`````````````0`!``$``0`!\`&
+M8`0@!"`$(`9@!\```````````!``$`#P__S_!P``````````````````````
+M`X`&0`0`!``$``9``X```````````!``$`#P__S_"``````````````@`"``
+M(``@`^`&8`0@!"`$(`9@`^```````````!``$`#P__S_"```````````````
+M`````````\`&8`0@!^`$``8@`<```````````!``$`#P__S_!```````````
+M``,`!``$``0`#P`$``0`!``$``0`!````````````!``$`#P__S_"```````
+M`````````````````^`&8`0@!"`$(`9@`^``(`)@`<```!``$`#P__S_"```
+M``````````0`!``$``0`!<`&(`0@!"`$(`0@!"```````````!``$`#P__S_
+M`P``````````````!```````!``$``0`!``$``0`!````````````!``$`#P
+M__S_`P``````````````!```````!``$``0`!``$``0`!``$``0`#````!``
+M$`#P__S_!P````````````0`!``$``0`!$`$@`4`!@`%``2`!$``````````
+M`!``$`#P__S_`P````````````0`!``$``0`!``$``0`!``$``0`!```````
+M`````!@`$`#P__S_#0`````````````````````````````%S@`&,0`$(0`$
+M(0`$(0`$(0`$(0`````````````````0`!``\/_\_P@`````````````````
+M``````7`!B`$(`0@!"`$(`0@```````````0`!``\/_\_P@`````````````
+M``````````/`!F`$(`0@!"`&8`/````````````0`!``\/_\_P@`````````
+M``````````````?`!F`$(`0@!"`&8`?`!``$``0````0`!``\/_\_P@`````
+M``````````````````/@!F`$(`0@!"`&8`/@`"``(``@```0`!``\/_\_P4`
+M``````````````````````6`!@`$``0`!``$``0````````````0`!``\/_\
+M_P<```````````````````````.`!$`$``.``$`$0`.````````````0`!``
+M\/_\_P4`````````````````!``$``\`!``$``0`!``$``<````````````0
+M`!``\/_\_P@```````````````````````0@!"`$(`0@!"`$8`.@````````
+M```0`!``\/_\_P<```````````````````````@@""`$0`1``H`"@`$`````
+M```````8`!``\/_\_PD`````````````````````````````"(@`"(@`!)``
+M!5``!5```B```B``````````````````$``0`/#__/\'````````````````
+M```````((`1``H`!``*`!$`((```````````$``0`/#__/\'````````````
+M```````````((`1`!$`"@`*``0`!``(``@`,````$``0`/#__/\'````````
+M```````````````'P`!``(`!``(`!``'P```````````$``0`/#__/\(````
+M````````````X`"``(``@`"``P``@`"``(``@`"``&``````$``0`/#__/\$
+M```````````````"``(``@`"``(``@`"``(``@`"``(``@`"````$``0`/#_
+M_/\(```````````````#@`"``(``@`"``&``@`"``(``@`"``P``````&``0
+M`/#__/\+``````````````````````````````````````.(``1P````````
+M`````````````````````!``$`#P__S_"``````````````````'\`00!!`$
+M$`00!!`$$`00!!`$$`?P`````!``$`#P__S_"```````````````````X`$0
+M`@`'X`(`!\`"``$0`.```````````!``$`#P__S_"``````````````````'
+M\`00!!`$$`00!!`$$`00!!`$$`?P`````!``$`#P__S_!```````````````
+M``````````````````````(``@`$`````````!``$`#P__S_"```````````
+M``!P`$``@`"``(``@`/``0`!``$``0`!``(`#@```!``$`#P__S_!P``````
+M``````````````````````````````*``H`%`````````!@`$`#P__S_#0``
+M```````````````````````````````````````````````"(@`"(@``````
+M```````````0`!``\/_\_P<``````````````````0`!``$`!\`!``$``0`!
+M``$``0`````````0`!``\/_\_P<``````````````````0`!``$`!\`!``$`
+M!\`!``$``0`````````0`!``\/_\_P<```````````````,`!(``````````
+M```````````````````````@`!``\/_\_Q$`````````````````````````
+M!@@```D0```)(```"2````9,8```DI```)*0``$2D``"#&``````````````
+M`````````!@`$`#P__S_"0`````````"0``!@``````#X``&$``$```&```#
+MX```,```$``$,``#X``````````````````0`!``\/_\_P4`````````````
+M``````````$``P`&``8``P`!```````````````8`!``\/_\_PX`````````
+M`````````````?^``Q``!!``!!``!!^`!!``!A```Q```?^`````````````
+M````$``0`/#__/\(``````````````````?P!!`$$`00!!`$$`00!!`$$`00
+M!_``````&``0`/#__/\*``````````)```&```````?X```0```@``!```!`
+M``"```$```(```?X`````````````````!``$`#P__S_"```````````````
+M```'\`00!!`$$`00!!`$$`00!!`$$`?P`````!``$`#P__S_"```````````
+M```````'\`00!!`$$`00!!`$$`00!!`$$`?P`````!``$`#P__S_!```````
+M```````````"``0`!````````````````````````````!``$`#P__S_!```
+M```````````````"``(`!````````````````````````````!``$`#P__S_
+M!P`````````````````"@`4`!0```````````````````````````!``$`#P
+M__S_!P`````````````````"@`*`!0```````````````````````````!``
+M$`#P__S_"````````````````````````8`#P`/``8``````````````````
+M`!``$`#P__S_!P``````````````````````````````#^``````````````
+M`````!@`$`#P__S_#0`````````````````````````````````````````/
+M_X`````````````````````````````0`!``\/_\_P<```````````````:`
+M!8`````````````````````````````````8`!``\/_\_PT`````````````
+M`````````Z0``3P``20``20`````````````````````````````````````
+M$``0`/#__/\'```````````````$@`,````#@`1`!``#@`!`!$`#@```````
+M````$``0`/#__/\%```````````````````````$``8``P`#``8`!```````
+M````````&``0`/#__/\-``````````````````````````````/>``9S``0A
+M``0_``0@``9Q``/>`````````````````!``$`#P__S_"```````````````
+M```'\`00!!`$$`00!!`$$`00!!`$$`?P`````!``$`#P__S_!P``````````
+M````!(`#````!\``0`"``0`"``0`!\```````````!``$`#P__S_!P``````
+M``````%````((`1`!$`"@`*``0`!``$``0```````````!``$`#P__S_"```
+M`````````````````````````````````````````````````!``$`#P__S_
+M!0`````````````````"``(````"``(``@`"``(``@```````````!``$`#P
+M__S_"```````````````````@`"``\`&H`2`!(`$@`:@`\``@`"``````!``
+M$`#P__S_"``````````````````!P`(@`@`"``>``@`"``(`#^``````````
+M`!``$`#P__S_"`````````````````````@@!\`$0`1`!$`'P`@@````````
+M`````!``$`#P__S_"``````````````````&,`(@`4`'<`"`!_``@`"``(``
+M`````````!``$`#P__S_!``````````````````"``(``@`"``(```````(`
+M`@`"``(``@```!``$`#P__S_!P`````````````````#P`0`!@`%@`3`!D`#
+M0`#``$`'@````````!``$`#P__S_!P`````````````````"@```````````
+M`````````````````````!@`$`#P__S_#0``````````````````````^``!
+MC``#=@`"@@`"@@`"@@`#=@`!C```^``````````````````0`!``\/_\_P8`
+M`````````````````X``0`/`!$`'P```!\`````````````````0`!``\/_\
+M_P@```````````````````````$@`V`&P`;``V`!(``````````````8`!``
+M\/_\_PL`````````````````````````````````````!_@```@```@```@`
+M````````````````````$``0`/#__/\%````````````````````````````
+M```'````````````````````&``0`/#__/\-``````````````````````#X
+M``&,``-V``)2``)B``)B``-6``&,``#X`````````````````!``$`#P__S_
+M!P``````````````!X```````````````````````````````````!``$`#P
+M__S_!P`````````````````#``2`!(`#`````````````````````````!@`
+M$`#P__S_"P``````````````````````````0```0``'_```0```0```````
+M```'_``````````````````0`!``\/_\_P4`````````````````!X``@`$`
+M`P`'@``````````````````````0`!``\/_\_P4`````````````````!X``
+M@`,``(`'@``````````````````````0`!``\/_\_P<```````````````$`
+M`@`````````````````````````````````0`!``\/_\_P@`````````````
+M``````````0@!"`$(`0@!"`$(`?P!``$``0````0`!``\/_\_P@`````````
+M`````````^`'H`>@!Z`#H`"@`*``H`"@`*`````````0`!``\/_\_P0`````
+M```````````````````````$``0````````````````````0`!``\/_\_P<`
+M`````````````````````````````````````````0``@`.````0`!``\/_\
+M_P4`````````````````!@`"``(``@`'```````````````````````0`!``
+M\/_\_P8``````````````````X`$0`1`!$`#@```!\`````````````````0
+M`!``\/_\_P@```````````````````````2`!L`#8`-@!L`$@```````````
+M```8`!``\/_\_PT`````````````````````!@P``@@``A```A``!R,``$4`
+M`$D``(^``8$`````````````````&``0`/#__/\-````````````````````
+M``8,``((``(0``(0``<O``!!``!"``"&``&/`````````````````!@`$`#P
+M__S_#0`````````````````````'C```B``#$```D``'HP``10``20``CX`!
+M@0`````````````````0`!``\/_\_P<``````````````````0`!`````0`!
+M``(`!``$0`.````````````8`!``\/_\_PD``````````0```(```````(``
+M`4```4```B```B``!_``!!``!!``"`@`````````````````&``0`/#__/\)
+M``````````"```$```````"```%```%```(@``(@``?P``00``00``@(````
+M`````````````!@`$`#P__S_"0`````````!@``"0```````@``!0``!0``"
+M(``"(``'\``$$``$$``("``````````````````8`!``\/_\_PD`````````
+M`T```L```````(```4```4```B```B``!_``!!``!!``"`@`````````````
+M````&``0`/#__/\)``````````````%```````"```%```%```(@``(@``?P
+M``00``00``@(`````````````````!@`$`#P__S_"0`````````!P``!0``!
+M0```@``!P``!0``!0``"(``#X``$$``$$``("``````````````````8`!``
+M\/_\_PT``````````````````````/\``*```2```2```C\``^``!"``!"``
+M"#\`````````````````&``0`/#__/\)``````````````````````'@``(0
+M``0```0```0```0```0```(0``'@``!````@``#@`````!``$`#P__S_"```
+M```````!``"````'X`0`!``$``?@!``$``0`!^```````````!``$`#P__S_
+M"```````````@`$````'X`0`!``$``?@!``$``0`!^```````````!``$`#P
+M__S_"``````````!@`)````'X`0`!``$``?@!``$``0`!^```````````!``
+M$`#P__S_"`````````````%````'X`0`!``$``?@!``$``0`!^``````````
+M`!``$`#P__S_`P`````````$``(````$``0`!``$``0`!``$``0`!```````
+M`````!``$`#P__S_`P`````````"``0````$``0`!``$``0`!``$``0`!```
+M`````````!``$`#P__S_`P`````````&``D````$``0`!``$``0`!``$``0`
+M!````````````!``$`#P__S_`P````````````4````$``0`!``$``0`!``$
+M``0`!````````````!@`$`#P__S_"@`````````````````````'X``$,``$
+M"``$"``/"``$"``$"``$,``'X``````````````````8`!``\/_\_PH`````
+M`````:```6``````!@@`!@@`!0@`!(@`!,@`!$@`!"@`!!@`!!@`````````
+M````````&``0`/#__/\*``````````"```!```````'@``(0``0(``0(``0(
+M``0(``0(``(0``'@`````````````````!@`$`#P__S_"@``````````0```
+M@``````!X``"$``$"``$"``$"``$"``$"``"$``!X``````````````````8
+M`!``\/_\_PH``````````,```2```````>```A``!`@`!`@`!`@`!`@`!`@`
+M`A```>``````````````````&``0`/#__/\*``````````&@``%@``````'@
+M``(0``0(``0(``0(``0(``0(``(0``'@`````````````````!@`$`#P__S_
+M"@``````````````H``````!X``"$``$"``$"``$"``$"``$"``"$``!X```
+M```````````````8`!``\/_\_PL``````````````````````````@@``1``
+M`*```$```*```1```@@`````````````````````&``0`/#__/\*````````
+M``````````````'H``(0``0H``1(``1(``2(``4(``(0``7@````````````
+M`````!@`$`#P__S_"@``````````@```0``````$"``$"``$"``$"``$"``$
+M"``$"``&&``!X``````````````````8`!``\/_\_PH``````````$```(``
+M````!`@`!`@`!`@`!`@`!`@`!`@`!`@`!A@``>``````````````````&``0
+M`/#__/\*``````````#```$@``````0(``0(``0(``0(``0(``0(``0(``88
+M``'@`````````````````!@`$`#P__S_"@``````````````H``````$"``$
+M"``$"``$"``$"``$"``$"``&&``!X``````````````````0`!``\/_\_P<`
+M`````````(`!````""`$0`1``H`"@`$``0`!``$````````````0`!``\/_\
+M_P@`````````````````!``'P`1@!"`$(`1@!\`$``0````````````0`!``
+M\/_\_P@````````````#@`1`!$`%@`4`!0`$P`0@!"`$(`7````````````0
+M`!``\/_\_P@```````````````$``(````/`!"``(`/@!"`$8`.@````````
+M```0`!``\/_\_P@```````````````"``0````/`!"``(`/@!"`$8`.@````
+M```````0`!``\/_\_P@```````````````&``D````/`!"``(`/@!"`$8`.@
+M```````````0`!``\/_\_P@```````````````-``L````/`!"``(`/@!"`$
+M8`.@```````````0`!``\/_\_P@``````````````````4````/`!"``(`/@
+M!"`$8`.@```````````0`!``\/_\_P@``````````,`!(`$@`,````/`!"``
+M(`/@!"`$8`.@```````````8`!``\/_\_PT`````````````````````````
+M`````]X`!#,``"$``_\`!"``!%$``XX`````````````````$``0`/#__/\'
+M```````````````````````#@`9`!``$``0`!D`#@`"``$`!P```$``0`/#_
+M_/\(```````````````!``"````#P`9@!"`'X`0`!B`!P```````````$``0
+M`/#__/\(````````````````@`$````#P`9@!"`'X`0`!B`!P```````````
+M$``0`/#__/\(```````````````!@`)````#P`9@!"`'X`0`!B`!P```````
+M````$``0`/#__/\(``````````````````%````#P`9@!"`'X`0`!B`!P```
+M````````$``0`/#__/\#```````````````$``(````$``0`!``$``0`!``$
+M````````````$``0`/#__/\#```````````````"``0````$``0`!``$``0`
+M!``$````````````$``0`/#__/\#```````````````&``D````$``0`!``$
+M``0`!``$````````````$``0`/#__/\#``````````````````4````$``0`
+M!``$``0`!``$````````````$``0`/#__/\(```````````````"0`.`!(`#
+MP`8@!"`$(`0@!F`#P```````````$``0`/#__/\(```````````````#0`+`
+M```%P`8@!"`$(`0@!"`$(```````````$``0`/#__/\(```````````````!
+M``"````#P`9@!"`$(`0@!F`#P```````````$``0`/#__/\(````````````
+M````@`$````#P`9@!"`$(`0@!F`#P```````````$``0`/#__/\(````````
+M```````!@`)````#P`9@!"`$(`0@!F`#P```````````$``0`/#__/\(````
+M```````````#0`+````#P`9@!"`$(`0@!F`#P```````````$``0`/#__/\(
+M``````````````````%````#P`9@!"`$(`0@!F`#P```````````&``0`/#_
+M_/\+``````````````````````````#```#```````?X``````#```#`````
+M`````````````````!``$`#P__S_"````````````````````````^`&8`2@
+M!2`%(`9@!\```````````!``$`#P__S_"````````````````0``@```!"`$
+M(`0@!"`$(`1@`Z```````````!``$`#P__S_"````````````````(`!````
+M!"`$(`0@!"`$(`1@`Z```````````!``$`#P__S_"````````````````8`"
+M0```!"`$(`0@!"`$(`1@`Z```````````!``$`#P__S_"```````````````
+M```!0```!"`$(`0@!"`$(`1@`Z```````````!``$`#P__S_!P``````````
+M`````(`!````""`$0`1``H`"@`$``0`"``(`#````!``$`#P__S_"```````
+M``````0`!``$``0`!\`&8`0@!"`$(`9@!\`$``0`!````!``$`#P__S_!P``
+E```````````````!0```""`$0`1``H`"@`$``0`"``(`#```````
+`
+end
